.github/workflows/tagged_master_release.yml

```diff
@@ -102,6 +99,35 @@ jobs:
           name: agave-win
           path: ./build/agave-win.zip
 
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+      - uses: ammaraskar/sphinx-action@master
+        with:
+          docs-folder: "docs/"
+      - uses: ammaraskar/sphinx-action@master
+        with:
+          docs-folder: "docs/"
+          pre-build-command: "apt-get update -y && apt-get install -y latexmk texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended"
+          build-command: "make latexpdf"
+      - uses: actions/upload-artifact@v1
+        with:
+          name: DocumentationHTML
+          path: docs/_build/html/
+      - uses: actions/upload-artifact@v1
+        with:
+          name: DocumentationPDF
+          path: docs/_build/latex/agave.pdf
+      - name: Publish Docs
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          BASE_BRANCH: master # The branch the action should deploy from.
+          BRANCH: gh-pages # The branch the action should deploy to.
```

Comment:
Is this the site you are expecting the workflow to deploy the docs to?
https://allen-cell-animated.github.io/agave/

Reply:
That should be the correct link yep!

Reply:
Yes.  Fingers crossed.

Reply:
Ok! Should this be linked in the README or is it generally understood on GitHub that this is where a dev should look for docs?

Reply:
Seconding adding a link to the README.

Reply:
This is not dev documentation but end-user documentation.  The README is really not for end-user documentation in this case.  It's worth adding a link there, sure, but really we'd want to link it from allencell.org and maybe put a link in the app itself as well.  I'm bundling the same documentation as a PDF with the app so end users will be able to see it that way too.